superset-frontend/src/components/ListView/types.ts

```diff
@@ -84,7 +82,7 @@ export interface FilterValue {
 export interface FetchDataConfig {
   pageIndex: number;
   pageSize: number;
-  sortBy: SortColumns;
+  sortBy: SortColumn[];
```

Comment:
Is there some reason why this is being changed from `SortColumns` to `SortColumn[]` here? While on this topic, I personally don't like these types of pluralization aliases, as it makes it seem like it's something more than just an array of the singular type. But if we want to refrain from making these types of sweeping changes during the 4.0 hardening phase, I'd recommend leaving this as-is for now, and following up later with a PR that removes the plural type.

Reply:
Personally I find following the "boy scout" approach is usually a good way of driving incremental change and reduces the pain of having to invest a chunk of time and effort into doing big bang changes. That's especially true if there are concerns over adequate test coverage for the "big bang" change, for example.

Though that's assuming there's already an agreed policy to remove aliases like these, otherwise the risk is different groups enacting different boy scout policies, which might be more of a concern in a large open source project like this one than I'm typically used to.